From 61a6eb94f08eb77a02c526c9e3795ea40caf1eeb Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Mon, 7 Jan 2019 08:51:28 +0800
Subject: [PATCH] arm64: dts: rockchip: vcc5v0_usb power on in suspend for
 rk3399pro evb

Some USB devices does not support power off during suspend, so the vbus
of the USB host port should be kept on during suspend.

Change-Id: Ia3b054f9c53c33b637a02629e8cb3f050a5158b6
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts | 3 ++-
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
index 863e12fc51fd..49fea10c1cb1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
@@ -593,7 +593,8 @@
 
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
+					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts
index 6e075b2e5579..793d58fff6b9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v11.dts
@@ -599,7 +599,8 @@
 
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
+					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
-- 
2.35.3

